components/molecule/fieldset/README.md

```diff
@@ -0,0 +1,61 @@
+# MoleculeFieldset
```

Comment:
The name is confusing to me. To me, the label and validation doesn't make it a fieldset.
I mean, a fieldset is a set of fields to me, with more that one input spot.